db/migrate/20210208201246_add_license_option_to_collection.rb

```diff
@@ -0,0 +1,10 @@
+class AddLicenseOptionToCollection < ActiveRecord::Migration[6.1]
+  def change
+    add_column :collections, :license_option, :string
+    Collection.all.each do |col|
+      col.update(license_option: col.user_can_set_license? ? 'depositor-selects' : 'required')
```

Comment:
I'm a little confused. At this point in the migration, won't all collections wind up getting 'required' since `user_can_set_license?` checks against the value of `license_option`? Which was just added in the prior line? I expected this line to use logic similar to https://github.com/sul-dlss/happy-heron/pull/1043/files#diff-ca0f3357f3c92b4b6b2f37ffd7dee610174fbe9a9a1d8c4a9354c985c73ca7a1L28 but maybe I am more confused than I realize :laughing:

Reply:
Oh, yeah, I made this before i made that other change.

Reply:
Fixed it.

Reply:
@jcoyne :beers: